packages/core/src/components/tag-input/tagInput.tsx

```diff
@@ -15,6 +15,17 @@ import * as Utils from "../../common/utils";
 import { Icon, IconName } from "../icon/icon";
 import { ITagProps, Tag } from "../tag/tag";
 
+/**
+ * A type reflecting the manner in which a `TagInput` value was added.
```

Comment:
nit: instead of "reflecting the manner ...", simpler language would be "The method in which a TagInput value was added."